File: sqlitejava/src/main/java/com/example/sqlitejava/DbApp.java
package com.example.sqlitejava;

import android.app.Application;
import android.arch.persistence.room.Room;

/**
 * Created by LENOVO on 15/05/2018.
 */

public class DbApp extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        AppDb db = Room.databaseBuilder(this,
                AppDb.class, "dbCar").build();
    }

}

File: app/src/main/java/com/example/lenovo/sqllite/DbApp.kt
package com.example.lenovo.sqllite

import android.app.Application
import android.arch.persistence.room.Room



/**
 * Created by LENOVO on 15/05/2018.
 */
class DbApp : Application(){

    val dbApp:AppDb by lazy{
        Room.databaseBuilder(this@DbApp,
                AppDb::class.java, "carDb").build()
    }

    override fun onCreate() {
        super.onCreate()

        dbApp
    }

}